app/src/main/java/com/github/swent/echo/data/model/EPFLSection.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.swent.echo.data.model
+
+interface EPFLSection
+
+enum class BachelorSection : EPFLSection {
```

Comment:
I think it would make sense to rather have an interface named just `Section` and have the class named something like `EPFLBcheorSection` etc. This way the interface is generic...